test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,19 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
-
-  test("returns Buzz when passed a multiple of 5", () => {});
-
-  test("returns FizzBuzz when passed a multiple 3 and 5", () => {});
-
-  test("returns the number when it isn't a multiple of 3 or 5", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toBe('Fizz');
```

Comment:
Nice, I would always have more than one test case though to make sure the answer isn't hard coded.